clients/src/main/java/org/apache/kafka/clients/admin/OffsetSpec.java

```diff
@@ -70,4 +72,21 @@ public static OffsetSpec maxTimestamp() {
         return new MaxTimestampSpec();
     }
 
+    /**
+     * Used to retrieve the offset with the local log start offset,
+     * log start offset is the offset of a log above which reads are guaranteed to be served
+     * from the disk of the leader broker, when Tiered Storage is not enabled, it behaves the same
+     * as the earliest timestamp
+     */
+    public static OffsetSpec earliestLocalSpec() {
+        return new EarliestLocalSpec();
+    }
+
+    /**
+     * Used to retrieve the offset with the highest offset of data stored in remote storage,
+     * and when Tiered Storage is not enabled, we won't return any offset (i.e. Unknown offset)
```

Comment:
nit: 

Used to retrieve ~~the offset with~~ the highest offset of data stored in remote storage[.]

Note: When tiered storage is not enabled, we ~~won't return any offset (i.e.~~  [will return] unknown offset.

Reply:
Thanks for the comments, already fixed them in the latest commit.